packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/vue/remotesignout.md

```diff
@@ -0,0 +1,48 @@
+After performing [local signout](/docs/guides/sign-users-out/vue/sign-out-of-your-app/), navigate the user's browser to the [OIDC logout page](https://developer.okta.com/docs/reference/api/oidc/#logout).
+
+This page clears the user's Okta session, and then redirects back to the `post_logout_redirect_uri` that is provided. This URI must be one of those listed in the `Logout redirect URI` section of your application's settings.
```

Comment:
I'm not sure if page is the right word here.
This page -> This endpoint maybe?

Reply:
No, it's actually a page, requiring the browser navigate to it rather than make an XHR request.